src/containers/App.js

```diff
@@ -31,7 +30,11 @@ class App extends Component {
 
   render() {
     const { isFetching, user } = this.props;
-    const isAuthenticated = user.email && user.state === 'active';
+    var isAuthenticated = false;
+    if(user) {
+      isAuthenticated = (user.email && user.state === 'active') ? true : false;
```

Comment:
(`? true : false`) this construction redundant, you can simplify
`isAuthenticated = user.email && user.state === 'active';`